### ISSUE TITLE
Ability to select which projects are shown in calendar/tasks

### ISSUE THREAD

**Issue by aproposnix** [nextcloud/deck#2312]
**Describe the solution you'd like**
Currently, with the new calendar integration, if the user enables the calendars, it will show all of the users calendars. It would be nice if the user had the option to select which projects were connected to calendar/tasks. 

**Describe alternatives you've considered**
No other alternative identified.

**Comment by jakobroehrl** [comment]
Hmm, but you can use the show/hide function (click on the calender name) in the calendar to show/hide the corresponding board.

**Comment by aproposnix** [comment]
> Hmm, but you can use the show/hide function (click on the calender name) in the calendar to show/hide the corresponding board.

Correct. But if you have a lot of projects you start to have a ton of visual clutter in both the calendar and tasks. Having to click on multiple calendars just to hide them becomes another obstacle. 

I also have projects boards that I am building up (for example, a product idea that may be developed one day). I don't necessarily need to have a calendar for that (only needed when starting project).

**Comment by bnjbvr** [comment]
Huge +1 for this one, since I'm sharing decks with my partner who is an iOS user. They wouldn't like to see these decks appear in the tasks list, but unfortunately there's no way to disable/hide CalDAV-synchronized task lists on iOS in the official task app, so it creates a lot of visual clutter there.

**Comment by deflektor-de** [comment]
> 
> 
> Hmm, but you can use the show/hide function (click on the calender name) in the calendar to show/hide the corresponding board.

You can also use Calendar's preference option "Show tasks in calendar" (under Settings & Import). This hides deck's calendars in the sidebar and removes the clutter. I know, this is just a solution for people, who don't use due dates in Deck (and Tasks) at all. 

Therefore I'm really supporting the implementation of this feature!

**Comment by sunjam** [comment]
Projects functionality deprecated in nc25.

- https://github.com/nextcloud/server/issues/28320
